component/motd.libsonnet

```diff
@@ -66,73 +51,67 @@ local motdRBAC =
     role_binding: role_binding,
   };
 
-local jobSpec = {
-  spec+: {
-    template+: {
-      spec+: {
-        containers_+: {
-          notification: kube.Container('sync-motd') {
-            image: '%(registry)s/%(repository)s:%(tag)s' % params.images.oc,
-            name: 'sync-motd',
-            workingDir: '/export',
-            command: [ '/scripts/motd_gen.sh' ],
-            env_+: {
-              HOME: '/export',
-            },
-            volumeMounts_+: {
-              export: {
-                mountPath: '/export',
-              },
-              scripts: {
-                mountPath: '/scripts',
+local jsonnetlib =
+  esp.jsonnetLibrary('motd', 'openshift-config') {
+    spec: {
+      data: {
+        'config.json': std.manifestJson({
+          motd: params.motd,
+        }),
+      },
+    },
+  };
+
+local jsonnetlib_ref = {
+  apiVersion: jsonnetlib.apiVersion,
+  kind: jsonnetlib.kind,
+  name: jsonnetlib.metadata.name,
+  namespace: jsonnetlib.metadata.namespace,
+};
+
+local managedresource =
+  esp.managedResource('motd-composer', 'openshift-config') {
+    metadata+: {
+      annotations: {
+        'syn.tools/description': |||
+          Composes the message of the day from manual entries in the syn hierarchy and active consolenotifications on the cluster.
+        |||,
+      },
+    },
+    spec: {
+      serviceAccountRef: { name: motdRBAC.motd_sa.metadata.name },
+      applyOptions: { force: true },
+      context: [
+        {
+          name: 'consolenotifications',
+          resource: {
+            apiVersion: 'console.openshift.io/v1',
+            kind: 'ConsoleNotification',
+            labelSelector: {
+              matchLabels: {
+                'appuio.io/notification': 'true',
               },
             },
           },
         },
-        volumes_+: {
-          export: {
-            emptyDir: {},
-          },
-          scripts: {
-            configMap: {
-              name: 'motd-gen',
-              defaultMode: std.parseOctal('0550'),
-            },
+      ],
+      triggers: [
+        {
+          name: 'consolenotifications',
```

Comment:
put on "sebi voice": it always triggers me when people wrongfully use plurals 😅🤣

Reply:
Only the reverse triggers me. Wrongful use of singulars for contexts. This is fine.